build/build.deps.xml

```diff
@@ -70,13 +74,27 @@
     </sequential>
   </macrodef>
 
+  <!-- Vendor Dependencies for umple.jar -->
+  <target name="deps.vendor" description="Install vendor dependencies">
```

Comment:
Yes.  I will keep going down this current line to get the build back up, and then we cal look at leveraging the Ivy to handle the work above